pkgs/development/libraries/spirv-headers/default.nix

```diff
@@ -1,13 +1,14 @@
 { stdenv, fetchFromGitHub, cmake }:
+
 stdenv.mkDerivation rec {
-  name = "spirv-headers-${version}";
-  version = "2019.1"; # spirv-tools version whose DEPS file calls for this commit
+  pname = "spirv-headers";
+  version = "1.4.1";
```

Comment:
> this is okay as long as header is more up-to-date than spirv-tools

Where is this guarantee made?

Reply:
LunarG seem to be adhering to SemVer, as long as a major version isn't publish, it should be compatible.

[It's about the vulkan-headers, but I think it's still relevant](https://www.lunarg.com/wp-content/uploads/2018/05/1.1-Compatibility-Statement_05_18.pdf)

Reply:
SPIRV-Headers is a different project developed by different people. Given that SPIRV-Tools goes out of its way to identify a *specific commit* that it relies on, not just a version number, I don't think we should presume to know better without an explicit guarantee from the maintainers.